### PR TITLE
Store & Sync LNURL info

### LINKED ISSUE #318: LNURL-pay: Store success action in separate table, linked to `SendSwap` by `payment_hash`
On a successful LNURL-pay, store the success action in a separate table.

### PR DESCRIPTION
**Pull request by dangeross**
This PR:
- [x] Stores the LNURL-pay/withdraw info in the payment details table
- [x] Decrypts the AES success action data once the preimage is received
- [x] Syncs the payment details table for Liquid and LNURL data

Needs PR https://github.com/breez/breez-sdk-greenlight/pull/1148

Fixes #318

### CODE REVIEW

File: lib/core/src/persist/mod.rs [hunk 1]
```diff
@@ -187,26 +194,117 @@ impl Persister {
...
+                "INSERT INTO payment_details (
+                tx_id,
+                destination,
+                description,
+                lnurl_info_json
+            )
+            VALUES (?, ?, ?, ?)
+            ON CONFLICT (tx_id)
+            DO UPDATE SET
+                {destination_update}
```
Comment (roeierez) [inline lib/core/src/persist/mod.rs:247]: Why don't we use the same coalesce statement for the destination like we do for lnurl info?

**Review by roeierez — APPROVED** [review]
LGTM. Added a small comment

File: lib/core/src/persist/mod.rs (hunk 1 above)
Comment (dangeross) [reply lib/core/src/persist/mod.rs:247]: The destination is not optional in the struct, so can't be set back to null

**Review by hydra-yse — APPROVED** [review]
LGTM

**Pull request merged**